### PR TITLE
Add top and bottom margin variables for left and right pullquotes

### LINKED ISSUE #191: Add specific top and bottom margin variables for left and right pullquotes

### PR DESCRIPTION
**Pull request by dannylonglegs**
fixes #191

### CODE REVIEW

File: assets/styles/variables/_specials.scss [hunk 1]
```diff
@@ -329,29 +329,53 @@ $pullquote-left-width: 45% !default;
...
+$pullquote-right-margin-bottom: $pullquote-margin-bottom !default;
+
 /// Text alignment for right pullquotes on elements with a class of `pullquote-right`.
 /// @type String
 /// @since 1.0.0
 $pullquote-right-align: $pullquote-align !default;
 
 /// Left margin for right pullquotes on elements with a class of `pullquote-right`.
 /// @type String | Map
+/// @since 1.1.0
```
Comment (greatislander) [inline assets/styles/variables/_specials.scss]: That's not correct, is it?

File: assets/styles/variables/_specials.scss [hunk 2]
```diff
@@ -329,29 +329,53 @@ $pullquote-left-width: 45% !default;
...
 $pullquote-right-width: 45% !default;
 
+/// Top margin for right pullquotes on elements with a class of `pullquote-right`.
+/// @type String | Map
+/// @since 1.1.0
+$pullquote-right-margin-top: $pullquote-margin-top !default;
+
 /// Right margin for right pullquotes on elements with a class of `pullquote-right`.
 /// @type String | Map
+/// @since 1.1.0
```
Comment (greatislander) [inline assets/styles/variables/_specials.scss]: That's not correct, is it?

**Comment by dannylonglegs** [comment]
I added maps to those variables on tuesday, shouldnt they be bumped up to
1.1.0?

**Comment by greatislander** [comment]
@dannylonglegs If the variable itself has been there since earlier, you should add a comment as seen in the example here:

http://docs.phpdoc.org/references/phpdoc/tags/since.html

So, something like:

`@since 1.1.0 Supports maps`

**Pull request merged**